pkgs/applications/blockchains/dogecoin/default.nix

```diff
@@ -0,0 +1,104 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkg-config,
+  autoreconfHook,
+  db5,
+  openssl,
+  boost,
+  zlib,
+  miniupnpc,
+  libevent,
+  protobuf,
+  qtbase ? null,
+  wrapQtAppsHook ? null,
+  qttools ? null,
+  qmake ? null,
+  qrencode,
+  withGui,
+  withUpnp ? false,
+  withUtils ? true,
+  withWallet ? true,
+  withZmq ? true,
+  zeromq,
+  util-linux ? null,
+  Cocoa ? null,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dogecoin" + lib.optionalString (!withGui) "d";
+  version = "1.14.8";
+
+  src = fetchFromGitHub {
+    owner = "dogecoin";
+    repo = "dogecoin";
+    rev = "v${version}";
+    sha256 = "sha256-Jmdxaona9bI9mw+WDGnrFU2ETgVTTQ7HHaWRodK/c4k=";
+  };
+
+  preConfigure = lib.optionalString withGui ''
+    export LRELEASE=${lib.getDev qttools}/bin/lrelease
+  '';
+
+  nativeBuildInputs =
+    [
+      pkg-config
+      autoreconfHook
+      util-linux
+    ]
+    ++ lib.optionals withGui [
+      wrapQtAppsHook
+      qttools
+    ];
+
+  buildInputs =
+    [
+      openssl
+      protobuf
+      boost
+      zlib
+      libevent
+    ]
+    ++ lib.optionals withGui [
+      qtbase
+      qrencode
+    ]
+    ++ lib.optionals withUpnp [ miniupnpc ]
+    ++ lib.optionals withWallet [ db5 ]
+    ++ lib.optionals withZmq [ zeromq ]
+    ++ lib.optionals stdenv.isDarwin [ Cocoa ];
+
+  configureFlags =
+    [
+      "--with-incompatible-bdb"
+      "--with-boost-libdir=${boost.out}/lib"
+    ]
+    ++ lib.optionals (!withGui) [ "--with-gui=no" ]
+    ++ lib.optionals (!withUpnp) [ "--without-miniupnpc" ]
+    ++ lib.optionals (!withUtils) [ "--without-utils" ]
+    ++ lib.optionals (!withWallet) [ "--disable-wallet" ]
+    ++ lib.optionals (!withZmq) [ "--disable-zmq" ];
+
+  enableParallelBuilding = true;
+
+  meta = with lib; {
+    description = "Wow, such coin, much shiba, very rich";
+    longDescription = ''
+      Dogecoin is a decentralized, peer-to-peer digital currency that
+      enables you to easily send money online. Think of it as "the
+      internet currency."
+      It is named after a famous Internet meme, the "Doge" - a Shiba Inu dog.
+    '';
+    homepage = "https://www.dogecoin.com/";
+    license = licenses.mit;
+    maintainers = with maintainers; [
+      edwtjo
+      offline
+      cleverca22
+      craigem
+    ];
```

Comment:
I think it'd be best to remove the maintainers who did not explicitly agree, even if this is just a revert.